users/faq-parts/general.rst

```diff
@@ -40,6 +42,10 @@ The following are *not* synchronized;
 
 -  Directory modification times (not preserved)
 -  Hard links (followed, not preserved)
+   - Following the link means that the linked file *is* synced, just as if it
```

Comment:
This seems a bit wordy for the middle of an otherwise concise bullet list. Is there really confusion about what not preserving hard links means, that doesn't stem from not understanding hard links to begin with?

Reply:
Maybe something like `Hard links (files synced as usual, hard links not preserved)` would cover it?